hydrography-approach/processing_scripts/filter_data/process_filter_nbi_bridges.py

```diff
@@ -31,7 +31,7 @@ def exclude_duplicate_bridges(df, output_duplicate_exclude_csv):
     return df
 
 
-def convert_to_gpkg(df, output_gpkg_file):
+def convert_to_gpkg(df, output_gpkg_file, logger):
```

Comment:
You can include exception handling here, also in other remaining functions as well

Reply:
I've added exception handling and type hints to the scripts.